apps/editors/helpers.py

```diff
@@ -750,12 +750,12 @@ def process_public(self):
         # Hold onto the status before we change it.
         status = self.addon.status
 
-        # Sign addon.
-        self.version.sign_files()
-
         self.set_files(amo.STATUS_PUBLIC, self.data['addon_files'],
                        copy_to_mirror=True)
 
+        # Sign addon.
+        self.version.sign_files()
```

Comment:
We only sign files that are prelim or fully reviewed. I've put the other calls to `sign_files` in the correct order in the other places in this file, but not that one. I'm now fixing this mistake ;)

I'll need to change that though for https://bugzilla.mozilla.org/show_bug.cgi?id=1156333, but in another PR